### ISSUE TITLE
Accessing a variable from a class is allowed to be a standalone line

### ISSUE THREAD

**Issue by GabeContra** [qorf/quorum-language#38]
### Example Code:

```
class Main
    action Main
        Other other
        other:name
    end
end
```

```
class Other
    public text name = "name"
end
```

The line other:name doesn't feel like it should be a legal line. This code compiles fine and does nothing but maybe it's an error? The reason seems to be that other:name is parsed as an action call but it's not really an action but maybe that's okay?

**Comment by andreas-stefik** [comment]
`class Main
    action Main
        Other other
        text value = other:name
        output value + other:name
        output value other:name
    end
end`

One reason why this might be good to change is it hits on two things:

1. It seems to be a real inconsistency in the language. Changing it fixes that.
2. It fixes cases like the above, which are clearly accidental, as it is pretty unlikely that a user is intending to throw the second value away.

**Comment by andreas-stefik** [comment]
This is now fixed and should come out with the next release. While I checked pretty carefully and it passes the test suite, this exposed a number of issues in the parser that did not appear to be correct. As such, a bit of dog fooding is probably in order for this particular change, even if it seems like it is working on my end.

**Comment by andreas-stefik** [comment]
Fix is in commit 442f1801d
